### PR TITLE
Upgrade Ur/Web version

### PR DESCRIPTION
**Pull request by achlipala**

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
Hi @achlipala - Thanks for your pr! Would you mind retargeting this to the round-14 branch?

**Comment by msmith-techempower** [comment]
@nbrady-techempower You realize today is your day off, right?

**Comment by nbrady-techempower** [comment]
@msmith-techempower I knooooow, but I'm working on my "weekend project" and couldn't resist :family:

File: frameworks/Ur/urweb/setup.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-VERSION=20160213
+VERSION=20160621
 COMPILER=${IROOT}/urweb
 
 RETCODE=$(fw_exists ${COMPILER}.installed)
```
Comment (msmith-techempower) [inline frameworks/Ur/urweb/setup.sh]: If `urweb` requires `mlton`, then in this file (somewhere) is the appropriate place to install it. Read: NOT `prerequisites.sh`.

**Comment by achlipala** [comment]
Replacing with #2133 

(Thanks for the help, @nbrady-techempower and @msmith-techempower.)

**Pull request closed without merging**